docs/policy.json.md

```diff
@@ -0,0 +1,248 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
+
+# Signature verification policy file format
+
+Signature verification policy files are used to specify policy, e.g. trusted keys,
+applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
```

Comment:
It does seem odd to have a default file location in a library.  I’m thinking that having each user of the library decide on its own, very likely different, location, would be much worse.

Of course a distributor shipping several such applications can patch them to use the same location—but that’s extra work and will still lead to fragmentation if various distributors don’t patch the same applications to use the same paths.

Also, the documentation of the format should be in this repo because that’s where the implementation is maintained, and then it’s a bit awkward to have a documentation for a file format which can’t tell the users where to use the format, or even how to find that out.

If an application wants to always use a different location, all it needs to do is call `signature.NewPolicyFromFile` instead of `signature.DefaultPolicy`, or at compile-time set `-ldflags '-X github.com/containers/image/signature.systemDefaultPolicyPath=/some/path'`. Or there can be a local-ish override via `types.SystemContext`  (all as of #49). There really are quite a few ways not to use the default, but I think it is very valuable to _have_ a default.


I do see the point that this is policy which might not be appropriate for this repo, and I’d be happy to change the PRs if that is the consensus.

If the specific value of the path is the reason for not liking it in a library, I personally really care very little what the path is.  A parallel discussion of that might happen in https://github.com/projectatomic/skopeo/pull/157 .